src/main/webapp/file-download-button-fragment.xhtml

```diff
@@ -347,10 +347,28 @@
     
     <!-- Request Access Button -->
     <ui:fragment rendered="#{!fileMetadata.datasetVersion.deaccessioned and !fileDownloadHelper.canDownloadFile(fileMetadata)
-                             and fileMetadata.dataFile.owner.fileAccessRequest}">
-        <!-- FILE PG -->
+                             and (fileMetadata.dataFile.owner.fileAccessRequest or not empty fileRequestAccessTools)}">
```

Comment:
Why or? Is the internal selection of whether file access requests can be made being overridden? If so, should some/all of the UI for managing terms be removed if external RA is enabled?